packages/electron-builder-squirrel-windows/src/SquirrelWindowsTarget.ts

```diff
@@ -98,24 +109,35 @@ export default class SquirrelWindowsTarget extends Target {
     const appInfo = packager.appInfo
     const projectUrl = await appInfo.computePackageUrl()
     const appName = this.appName
+    // If not specified will use the Squirrel.Windows that is shipped with electron-installer(https://github.com/electron/windows-installer/tree/main/vendor)
+    // After https://github.com/electron-userland/electron-builder-binaries/pull/56 merged, will add `electron-builder-binaries` to get the latest version of squirrel.
+    const vendorDirectory = this.options.customSquirrelExePath
     const options: SquirrelOptions = {
       name: appName,
-      productName: this.options.name || appInfo.productName,
       appId: this.options.useAppIdAsId ? appInfo.id : appName,
       version: appInfo.version,
       description: appInfo.description,
-      // better to explicitly set to empty string, to avoid any nugget errors
+      exe: `${this.packager.platformSpecificBuildOptions.executableName || this.options.name || appInfo.productName}.exe`,
       authors: appInfo.companyName || "",
       iconUrl,
       extraMetadataSpecs: projectUrl == null ? null : `\n    <projectUrl>${projectUrl}</projectUrl>`,
       copyright: appInfo.copyright,
       packageCompressionLevel: parseInt((process.env.ELECTRON_BUILDER_COMPRESSION_LEVEL || packager.compression === "store" ? 0 : 9) as any, 10),
-      vendorPath: await getBinFromUrl("Squirrel.Windows", "1.9.0", "zJHk4CMATM7jHJ2ojRH1n3LkOnaIezDk5FAzJmlSEQSiEdRuB4GGLCegLDtsRCakfHIVfKh3ysJHLjynPkXwhQ=="),
+      vendorDirectory,
+      nuspecTemplate: path.join(__dirname, "..", "template.nuspectemplate"),
       ...(this.options as any),
     }
 
+    if (await (await packager.signingManager.value).cscInfo.value) {
+      options.windowsSign = {
+        hookFunction: async (file: string) => {
+          await packager.sign(file)
+        },
+      }
+    }
+
     if (isEmptyOrSpaces(options.description)) {
-      options.description = options.productName
+      options.description = this.options.name || appInfo.productName
```

Comment:
Why this change? This may have unintended side effects

Reply:
The original code was like this:
```
options.description = options.productName;
options.productName = this.options.name || appInfo.productName;
```

After this MR, there is no productName in the SquirrelOptions, so it should now be directly changed to:

```
options.description = this.options.name || appInfo.productName;
```